operations/host_spawn.go

```diff
@@ -212,15 +213,20 @@ func hostModify() cli.Command {
 			},
 			cli.StringFlag{
 				Name:  addSSHKeyFlag,
-				Usage: "add user defined public key corresponding the `KEY_NAME` to the host's authorized_keys",
+				Usage: "add `PUBLIC_KEY` to the host's authorized_keys",
```

Comment:
Can you clarify that this is the public key plaintext from the file, and not the public key file itself?

Reply:
I opted to change this option to read in the key from a file.
The help text now reads
> add public key from local file `PATH` to the host's authorized_keys